stingray/utils.py

```diff
@@ -259,7 +263,7 @@ def rebin_data(x, y, dx_new, yerr=None, method='sum', dx=None):
 
     dx_var = np.var(dx_old) / np.mean(dx_old)
 
-    if np.size(dx_old) == 1 or dx_var < 1e-6:
+    if dx_old.size == 1 or dx_var < 1e-6:
```

Comment:
Same here, if the user has put a list into `dx` rather than a numpy array, this will break.

Reply:
Done